src/blockchains/erc20/erc20_types.ts

```diff
@@ -0,0 +1,14 @@
+export interface ERC20Transfer {
```

Comment:
TS wont verify types in runtime so for objects that we send to Kafka I suggest to use something like `zod` to verify them

Reply:
Sounds good, but this would require going further in type enforcing that this PR is doing. Let's leave this for another iteration.